pkg/keccak/contrib/Modes/keccak1600.h

```diff
@@ -0,0 +1,211 @@
+/*
+
+Implementation by the Keccak, Keyak and Ketje Teams, namely, Guido Bertoni,
+Joan Daemen, Michaël Peeters, Gilles Van Assche and Ronny Van Keer, hereby
+denoted as "the implementer".
+
+For more information, feedback or questions, please refer to our websites:
+http://keccak.noekeon.org/
+http://keyak.noekeon.org/
+http://ketje.noekeon.org/
+
+To the extent possible under law, the implementer has waived all copyright
+and related or neighboring rights to the source code in this file.
+http://creativecommons.org/publicdomain/zero/1.0/
+
+This is free and unencumbered software released into the public domain.
+
+Anyone is free to copy, modify, publish, use, compile, sell, or
+distribute this software, either in source code form or as a compiled
+binary, for any purpose, commercial or non-commercial, and by any
+means.
+
+In jurisdictions that recognize copyright laws, the author or authors
+of this software dedicate any and all copyright interest in the
+software to the public domain. We make this dedication for the benefit
+of the public at large and to the detriment of our heirs and
+successors. We intend this dedication to be an overt act of
+relinquishment in perpetuity of all present and future rights to this
+software under copyright law.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+OTHER DEALINGS IN THE SOFTWARE.
+
+For more information, please refer to <http://unlicense.org/>
+*/
+
+#ifndef KECCAK1600_H
+#define KECCAK1600_H
+
+#define KeccakP200_excluded
+#define KeccakP400_excluded
+
+#ifndef KeccakP1600_excluded
+
+#include "keccak_defs.h"
+#include "KeccakHash.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   A struct for an instance of Keccak-1600, containing state and parameters.
+ */
+#define keccak1600hash_instance Keccak_HashInstance
```

Comment:
Since the `#define` is basically only there for consistency with RIOT's naming conventions, I don't think there was another reason than "it felt more right". Does your comment apply only to `keccak1600hash_instance` or also to `fixed_output_length` and `delimited_suffix` below?
Also, I don't see any `#defines` in `keccak800.h`.

Reply:
Nevermind, I just realized that this:

> (same comments for keccak800)

was referring to the comment on the instance, not the typedef. Can't believe it took me so long.

I've simply replaced all `#defines` with `typedefs` and renamed `keccak1600hash_instance` and `keccak800hash_instance` to `keccak1600_t` and `keccak800_t`, respectively, so I'm marking this as resolved.